src/game/common/gameengine.cpp

```diff
@@ -140,11 +140,13 @@ void GameEngine::Init(int argc, char *argv[])
         "Data/INI/Default/Roads.ini",
         "Data/INI/Roads.ini");
     Init_Subsystem(g_theGlobalLanguage, "TheGlobalLanguageData", new GlobalLanguage);
+#ifdef GAME_DLL
```

Comment:
This means it is compiled out in Standalone. I assume this is not what is wanted?

Reply:
Ok put a comment then explaining situation.